coverage_ai/runner.py

```
@@ -1,41 +1,35 @@
 import subprocess
 import time
 
-from coverage_ai.settings.config_loader import get_settings
-
 
 class Runner:
     @staticmethod
-    def run_command(command, cwd=None):
+    def run_command(command: str, max_run_time_sec: int, cwd: str = None):
```

Ruff (0.13.3)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)